homeassistant/components/iskra/strings.json

```diff
@@ -0,0 +1,77 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Configure Iskra Device",
+        "description": "Enter the IP address of your Iskra Device and select protocol.",
+        "data": {
+          "host": "Host"
+        }
+      },
+      "authentication": {
+        "title": "Configure Rest API Credentials",
+        "description": "Enter username and password",
+        "data": {
+          "username": "Username",
+          "password": "Password"
+        }
+      },
+      "modbus_tcp": {
+        "title": "Configure Modbus TCP",
+        "description": "Enter Modbus TCP port and device's Modbus address.",
+        "data": {
+          "port": "Port",
+          "address": "Modbus address"
```

Comment:
What's a modbus address?

Reply:
It's a device's address inside Modbus network it must be unique inside network, You could think of it like a simplified IP address, its a 1 byte number. User can set it on device and it's required to communicate with device with Modbus (part of modbus packet)

Reply:
How can the user know what to fill in?

Reply:
This is set on device, user can navigate through user menu and set this settings but by default port is 10001 and modbus address  33

Reply:
Would it make sense to add a `data_description` thing where we explain where the user can find these values?